webui/react/src/pages/TrialDetails/TrialDetailsHyperparameters.tsx

```diff
@@ -58,19 +59,25 @@ const TrialDetailsHyperparameters: React.FC<Props> = ({ trial, pageRef }: Props)
 
   return (
     <div className={css.base}>
-      <Spinner conditionalRender spinning={!trial}>
-        <InteractiveTable
-          columns={columns}
-          containerRef={pageRef}
-          dataSource={dataSource}
-          pagination={false}
-          rowClassName={defaultRowClassName({ clickable: false })}
-          rowKey="hyperparameter"
-          settings={settings as InteractiveTableSettings}
-          showSorterTooltip={false}
-          size="small"
-          updateSettings={updateSettings as UpdateSettings<InteractiveTableSettings>}
-        />
+      <Spinner spinning={!trial}>
+        {
+          trial
+            ? (
+              <InteractiveTable
+                columns={columns}
+                containerRef={pageRef}
+                dataSource={dataSource}
+                pagination={false}
+                rowClassName={defaultRowClassName({ clickable: false })}
+                rowKey="hyperparameter"
+                settings={settings as InteractiveTableSettings}
+                showSorterTooltip={false}
+                size="small"
+                updateSettings={updateSettings as UpdateSettings<InteractiveTableSettings>}
+              />
+            )
+            : <SkeletonTable columns={2} />
```

Comment:
Why put the `SkeletonTable` in the `ResponsiveTable` component, but not in the `InteractiveTable` component (along with `loading` prop)?

Makes things kinda inconsistent